benchmarks/asv.conf.json

```diff
@@ -11,12 +11,16 @@
 
     // The URL of the source code repository for the project being
     // benchmarked
-    "repo": "https://github.com/scipy/scipy.git",
+    //"repo": "https://github.com/scipy/scipy.git",
+    //"repo": "/home/sgubian/tmp/scipy",
```

Comment:
Sure! Sorry about that. I forgot to remove it when trying to run the benchmark as  it gets source code from specified branches.